assets/src/components/ladderPage.tsx

```diff
@@ -78,6 +82,14 @@ const LadderPage = (): ReactElement<HTMLDivElement> => {
                 route={vehicleRoute(routes, selectedVehicleOrGhost)}
               />
             )}
+            {vehicleAndRouteForNotification && (
+              <PropertiesPanel
```

Comment:
Is it a problem that this could draw two PropertiesPanels? Should we have something like
```jsx
vehicleForTheVPP = selectedVehicleOrGhost || vehicleAndRouteForNotification
vehicleForTheVPP && <PropertiesPanel/>
```
?

Reply:
It shouldn't be able to render two PPs, since `selectedVehicleOrGhost` and `vehicleAndRouteForNotification` are only mutated through reducers that ensure that they're not both set at once. But that said, never hurts to check.

Actually here's an idea: what if we go with what you're suggesting here, but we also add something like:

```
if(selectedVehicleOrGhost && vehicleAndRouteForNotification) {
  sendWarningToSentry("supposedly impossible state happening");
}
```